capstone/src/main/java/com/google/univiz/SuggestionDataApi.java

```diff
@@ -7,9 +7,6 @@
  * client query
  */
 public interface SuggestionDataApi {
-  /**
-   * Takes a college name and returns a list of SuggestionData objects, where each object is unique
-   * to each college name provided in the first list
-   */
+  /** Takes a college name and returns a list of SuggestionData objects */
```

Comment:
Might be good to be clear on what `collegeName` can be:

```suggestion
  /** Takes a fragment of a college name or a complete college name and returns a list of SuggestionData objects */
```